controller/src/main/java/org/jboss/as/controller/descriptions/ModelDescriptionConstants.java

```diff
@@ -162,6 +163,7 @@
     public static final String FILE = "file";
     public static final String FILE_HANDLER = "file-handler";
     public static final String FILTER = "filter";
+    //public final static String FILTER_CONJUNCT = "conjuinct";
```

Comment:
Can you remove this? Or fix the spelling of "conjuinct" so if it gets uncommented later it doesn't slip in misspelled.